voluncheer/requirements.txt

```diff
@@ -4,4 +4,6 @@ django_debug_toolbar==3.8.1
 Django==3.2.8
 pytz==2022.7.1
 sqlparse==0.4.3
-pysqlite3-binary
+#Not installable by pip
+#pysqlite3-binary
```

Comment:
I was able to install this via pip:

pysqlite3-binary==0.5.0

Reply:
I will put it back in my later branch pull request.

Reply:
Actually in light of moving toward PostgresSQL you can just delete it.

Reply:
Will remove in the future when that happens.

Reply:
What purpose does it serve as a comment? There is no sense in leaving it commented out. If not having it is an issue we add it back.

Reply:
Sorry, typo. Fixed.